artifactory/commands/utils/configfile.go

```diff
@@ -298,7 +300,7 @@ func getIncludeExcludePatterns(patternType string) string {
 	var patterns []string
 	patternNum := 1
 	for {
-		newPattern := AskString("", strings.Title(patternType)+" pattern "+strconv.Itoa(patternNum)+" (leave empty to continue):", true, false)
+		newPattern := AskString("", cases.Title(language.Und, cases.NoLower).String(patternType)+" pattern "+strconv.Itoa(patternNum)+" (leave empty to continue):", true, false)
```

Comment:
What is the reason for this change?

Reply:
We updated CLI-Core to Go v1.18, this caused Static Analysis failure, because 'strings.Title()' is now deprecated in Go v1.18, and 'cases' replace it:
https://github.com/jfrog/jfrog-cli-core/actions/runs/2942074856